src/components/datepicker/datepicker.js

```diff
@@ -158,18 +158,18 @@ Datepicker.defaults = {
       'December',
     ],
     monthNamesShort: [
-      'January',
-      'February',
-      'March',
-      'April',
+      'Jan',
```

Comment:
this change should be mentioned in the migration doc. This would probably require some translation work